### DISCUSSION TITLE
멀티플레이 테스트

### DISCUSSION THREAD

**Discussion by ikaxani** [naverz/zepeto-studio-kor#552] [5. Q&A]
유니티 에디터 말고 publish후 제페토스튜디오에 등록한 테스트링크로 
멀티플레이 테스트를 할수있나요?

**Comment by x4cf** [comment]
가능합니다

**Reply by ikaxani** [reply]
감사합니다. 멀티플레이예제 프로젝트 파일로 테스트 해봤는데, 스튜디오 링크 접속으로 멀티가 안되는 것 같아서요. 
혹시 추가해야하는 부분이 따로 있나요?

**Reply by Spice8876** [reply]
https://studio.zepeto.me/guides/preparing-for-world-creating
월드 ID를 프로젝트에 연결 해야할거에요

**Reply by ikaxani** [reply]
답변 감사합니다.
월드ID-제페토스튜디오 콘솔 등록, 연결과정 정상적으로 진행했습니다, 
유니티에디터, QR코드 생성접속으로는 멀티테스트가 되는데,...
퍼블리시->패키징파일 등록 후 생성되는 테스트링크로 접속한 테스터인원으로 멀티플레이가 되지 않아서요.
또 확인해야 할 부분이 있을까요?

**Reply by Zepeto-TechEvangelist** [reply]
만약 다른 설정에 문제가 없다면 아래의 원인이 있을 수 있겠습니다.
테스트 인원 등록에 약 10분 내외의 시간이 소요되므로 바로 테스트를 하면 안될 수 있습니다.

**Reply by ikaxani** [reply]
감사합니다. 확인된 멀티플레이 테스트가 가능한 방법이
에디터에서 QR생성 후 QR접속을 한번 해야,
스튜디오 테스트링크로 접속 시 멀티플레이테스트가 되는 것 같습니다.
링크접속으로 바로 멀티테스트가 되었으면 좋겠는데 잘 모르겠네요...ㅜ
